inc/Engine/Media/PreloadFonts/Context/Context.php

```diff
@@ -47,4 +90,22 @@ public function is_mobile_allowed(): bool {
 			&& $this->options->get( 'do_caching_mobile_files', 0 )
 			&& wp_is_mobile();
 	}
+
+	/**
+	 * Get array of fonts to be excluded.
+	 *
+	 * @return array
+	 */
+	public function get_exclusions(): array {
+		$lists      = $this->data_manager->get_lists();
+		$lists      = isset( $lists->delay_js_exclusions ) ? $lists->delay_js_exclusions : [];
```

Comment:
```suggestion
		$lists      = $lists->delay_js_exclusions ?? [];
```

I believe we need to change the `delay_js_exclusions` to be the one mentioned in rucss-backend PR, correct?